### PR TITLE
Nerfs memes that were a little too dank.

### PR DESCRIPTION
**Pull request by Iamgoofball**
see changes

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/fox.dm [hunk 1]
```diff
@@ -21,5 +21,5 @@
 //Captain fox
 /mob/living/simple_animal/pet/fox/Renault
 	name = "Renault"
-	desc = "Renault, the Captain's trustworthy fox. I wonder what it says?"
```
Comment (Iamgoofball) [inline code/modules/mob/living/simple_animal/friendly/fox.dm]: okay uhm the desc is still there in the file

ffs github

**Comment by Alek2ander** [comment]
:-1: The dankness is our life

**Comment by killerfgs** [comment]
you cant take our dank memes away

**Comment by WJohn** [comment]
:-1:

**Comment by Thunder12345** [comment]
:-1: Dankest memes :-1:

**Comment by Dorsisdwarf** [comment]
0/10 you cannot the memes

**Comment by Thunder12345** [comment]
But really, why remove the whole of Renault's desc?

**Comment by RandomMarine** [comment]
He didn't, it says "Renault, the Captain's trustworthy fox." Github isn't showing the diff right.

**Comment by Thunder12345** [comment]
Pfffffft fucking Git

**Comment by Cheridan** [comment]
>the only memes in the game should be mine -- Iamgoofball

**Comment by Iamgoofball** [comment]
>the only conspiracies in the game should be mine -- Cheridan

**Pull request closed without merging**